opennebula/python/tar/slipstream_opennebula/OpenNebulaClientCloud.py

```diff
@@ -148,6 +148,15 @@ def _set_disks(self, image_id):
             raise "Something wrong with image ID : %s!" % image_id
         return 'DISK = [ IMAGE_ID  = %d ]' % img_id
 
+    def _set_additionnal_disks(self, disk_size_gb):
+        if disk_size_gb is None:
+            return ''
+        try:
+            disk_size_mb = float(disk_size_gb) * 1024
+        except:
+            raise "Something wrong with additionnal disk size : %s!" % disk_size_gb
+        return 'DISK = [ FORMAT = "ext4", SIZE="%d", TYPE="fs" ]' % disk_size_mb
```

Comment:
It's better to do it like this:
```python
return 'DISK = [ FORMAT = "ext4", SIZE="%d", TYPE="fs" ]' % (disk_size_mb,)
```
Because if at some point the variable `disk_size_mb` is an iterable it will fail with a non meaningful error message.
This is one of the argument against this syntax and also one of the reason why the `format()` method was added.
```python
return 'DISK = [ FORMAT = "ext4", SIZE="{}", TYPE="fs" ]'.format(disk_size_mb)
```
The following website has good examples of how to convert code to the `format` method: https://pyformat.info/

BTW I see that this also apply to others part of this code.